Magento_ConfigurableProduct/templates/product/price/final_price.phtml

```diff
@@ -62,14 +62,14 @@ $schema          = $block->getZone() == 'item_view';
 <?php if ($block->showMinimalPrice()): ?>
     <?php if ($block->getUseLinkForAsLowAs()): ?>
         <a
-            href="<?= /* @escapeNotVerified */ $block->getSaleableItem()->getProductUrl() ?>"
+            href="<?= /* @noEscape */ $block->getSaleableItem()->getProductUrl() ?>"
```

Comment:
as in original file we should use here
`<?= $block->escapeUrl($block->getSaleableItem()->getProductUrl()) ?>`